### ISSUE TITLE
Select

### ISSUE THREAD

**Issue by halimahLDN** [Greater-London-Authority/ldn-viz-tools#746]
- [ ] Clicking the label should focus the Select element:
   - [ ] Use a `for`/`id` pairing to associate input with a label element.

- [x] Disabled story does not meet minimum colour contrast ratio thresholds:
![image](https://github.com/user-attachments/assets/0ed52324-d560-40e7-8c92-e34d14a4e10c)
![image](https://github.com/user-attachments/assets/6e99aec3-f41c-4278-8962-a772ab998fb3)
